lib/routes/users.js

```diff
@@ -0,0 +1,225 @@
+'use strict';
+
+/**
+ * @fileoverview Login routing
+ */
+const config = require('config');
+const types = require('punchcard-content-types');
+const multipart = require('connect-multiparty');
+const multipartMiddleware = multipart();
+const bcrypt = require('bcrypt-nodejs');
+
+const acl = require('../auth/acl');
+const getUserId = require('../auth/acl').getUserId;
```

Comment:
Why do this instead of `acl.getUserId`? And can you use a better name, like `user`?